cpp/velox/substrait/TypeUtils.h

```diff
@@ -97,6 +97,9 @@ struct RangeTraits<TypeKind::DATE> {
 
 template <>
 struct RangeTraits<TypeKind::HUGEINT> {
+  // TODO
+  // using RangeType = common::BigintRange;
+  // using MultiRangeType = common::BigintMultiRange;
   using NativeType = int128_t;
```

Comment:
> Add process for SubstraitToVeloxPlanConverter::setFilterMap in case of TypeKind::HUGEINT. The validation will fail if the field is Decimal<4,20>.

If the filter pushdown for HUGEINT is not ready, can we remove this change and add it with its support?

Reply:
the validation will fail if there is long decimal column without filter, so this pr resolve this problem.